src/index.css

```diff
@@ -1,3 +1,202 @@
 body {
-  font-family: 'Courier New', Courier, monospace;
+  font-family: 'Montserrat', 'Times New Roman', Times, serif;
+  background-color: #f5f6fa;
+}
+
+.book-container {
+  display: flex;
+  flex-direction: column;
+  padding: 1.681279vh 7.24vw 2.32128vh 7.32vh;
+}
+
+.input-container {
+  display: flex;
+  flex-direction: column;
+  border-top: 2px solid #e8e8e8;
+  padding-bottom: 20vh;
+}
+
+.input-h2 {
+  font-size: 1.25rem;
+  font-weight: bolder;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.input-form {
+  display: flex;
+  justify-content: space-between;
+  gap: 2vw;
+}
+
+.input-title {
+  width: 41.5rem;
+  height: 1.5rem;
+}
+
+.input-category {
+  width: 17.813rem;
+  height: 1.9rem;
+}
+
+.input-button {
+  width: 11.5rem;
+  height: 1.9rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  font-size: 0.813rem;
+  font-weight: bold;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+  border-color: #0290ff;
+}
+
+.list-container {
+  display: flex;
+  flex-direction: column;
+  gap: 2vh;
+  padding-left: 0;
+}
+
+.box {
+  display: flex;
+  justify-content: space-between;
+  border: 1px solid #e8e8e8;
+  background-color: #fff;
+  padding: 2vh 2vw;
+}
+
+.list-left-comp,
+.list-right-comp {
+  list-style: none;
+}
+
+.list-left-category {
+  opacity: 0.5;
+  font-size: 0.875rem;
+  font-weight: bold;
+  color: #121212;
+}
+
+.list-left-title {
+  font-size: 1.375rem;
+  font-family: Roboto Slab, Arial, sans-serif;
```

Comment:
- [ ] It will be nice to create a class for this property for easy re-use since you have multiple declarations for this style.

Reply:
Hello @wandji20 can you explain what you mean by ignoring all files that are not relevant to run this project(eg .lintrc, .stylelintrc.json, .hintrc, .eslintrc.json etc). Do you mean that I have to disable linters ?

Reply:
Hi @aliabbani,
The files I mentioned are required to run Linters locally. These you can ignore, yet keep in mind that your GitFlow configuration is a project requirement.

Reply:
Alright thank you